src/contact.ts

```diff
@@ -55,8 +55,10 @@ export enum Gender {
 }
 
 export type ContactQueryFilter = {
-  name?: string | RegExp
+  name?:   string | RegExp
+  // remark should be deprecated
   remark?: string | RegExp
```

Comment:
ok, I guess I can use function `Contact.find(query: ContactQueryFilter | {remark: string})` instead, is it right?